pkg/connectors/datacatalog/openapiclient/.openapi-generator/FILES

```diff
@@ -0,0 +1,13 @@
+.gitignore
+.openapi-generator-ignore
+.travis.yml
+README.md
+api/openapi.yaml
```

Comment:
this is also not needed. We generate openapiclient only once unlike base which can be generated multiple times. So it is better to remove unnecessary files created during code generation. Thanks.

Reply:
Done